lambda/es-proxy-layer/lib/esbodybuilder.js

```diff
@@ -104,11 +104,40 @@ function build_query(params) {
       if (_.get(params, 'score_answer_field')) {
         query = query.orQuery('match', 'a', params.question);
       }
-      query = query.orQuery('match', 't', _.get(params, 'topic', ''))
+      let topic = _.get(params, 'topic');
+      if (topic) {
+        query = query.orQuery('match', 't', topic);
+      } else {
+        // no topic - query prefers answers with empty/missing topic field for predicable response
+        // NOTE: will not work in Kendra FAQ mode since we have no equivalent Kendra query
```

Comment:
- Can we add a description of this feature and a note to the Implementation Guide. So, customers are aware that this won't work in Kendra FAQ mode.

Reply:
> description of this feature and a note to the Implementation Guide
Great idea.. No tim ethis week though, but let's discuss this next week, and the process for updating the IG.